src/components/WorkOrder/MobileWorkingWorkOrderView.js

```diff
@@ -118,29 +122,49 @@ const MobileWorkingWorkOrderView = ({ workOrderReference }) => {
   const onWorkOrderCompleteSubmit = async (data) => {
     setLoading(true)
 
+    let followOnRequest = null
+
+    if (data['followOnStatus'] === 'furtherWorkRequired') {
```

Comment:
Why not `data.followOnStatus` with dot notation etc, and is there any risk of null/undefined here that should be handled?

Reply:
The backend request object doesnt like null. The object must either be populated, or not present in the request

Reply:
Ill change to dot notation if ther are other changes. Im not worth waiting another 20 mins for the tests to run